pyrightconfig.json

```diff
@@ -13,5 +13,5 @@
         // Documentation configuration files do not require type checking
         "docs/conf.py"
     ],
-    "pythonVersion": "3.10"
+    "pythonVersion": "3.13"
```

Comment:
This should the oldest version supported, not the newest

Reply:
Done in [51379dd](https://github.com/py-mine/mcstatus/pull/929/commits/51379dd13e4a404717fd9565934a6a9ae99414b3).

I've also mentioned this as another point in the PR's description, which also explains why it was set to the latest value before.